jobs/e2e-test-cycloud.groovy

```diff
@@ -152,30 +154,74 @@ def runAllTestClients() {
    for (i = 0; i < NUM_WORKER_MACHINES; i++) {
       def workerId = i;  // avoid scoping problems
       jobs["e2e-test-${workerId}"] = {
-         swallowExceptions({
```

Comment:
Do you not want to swallow exceptions anymore?

Reply:
I think nothing runs because you haven't defined `onException` in this script.  You should copy it from webapp-test if you want it.  In general, you should think about what you want to actually happen if one of the workers throws an exception for some reason.  In webapp-test, we depend on analysis of the junit xml files to figure out what worked and what didn't, so swallowing exceptions is correct.  But it's not clear to me if that's what you're doing here or not.